scania-digital-design-system/tegel
# chore: move storybook to core folder

**Describe pull-request**  
Moves the storybook installation to core so it doesn't span multiple packages.
This also fixes an issue where code syntax highlighting were not working in mdx files.

**Solving issue**  
Somewhat related to https://tegel.atlassian.net/browse/DTS-1595 

**How to test**  
Open the deploy preview and check that everything runs correctly, add ons are showing up, etc.

**Additional context**  
Since the storybook_static folder will now be generated inside core/ the netlify configuration needs to change for this to work.
